zpls_echograms/zpls_echogram.py

```diff
@@ -2,131 +2,189 @@
 from echopype.convert import Convert
 from echopype.process import Process
 from pandas.plotting import register_matplotlib_converters
+from PIL import Image
 
+warnings.filterwarnings('ignore', category=FutureWarning)
 register_matplotlib_converters()
-subsite_config = {
+site_config = {
     'CE01ISSM': {
-        'long_name': 'Oregon Inshore Surface Mooring',
+        'long_name': 'Coastal Endurance, Oregon Inshore Surface Mooring',
         'tilt_correction': 15,
         'colorbar_range': [-90, -50],
         'vertical_range': [0, 25],
         'deployed_depth': 25
     },
+    'CE02SHBP': {
+        'long_name': 'Coastal Endurance, Oregon Shelf Cabled Benthic Experiment Package',
+        'tilt_correction': 0,
+        'colorbar_range': [-90, -50],
+        'vertical_range': [0, 80],
+        'deployed_depth': 80
+    },
+    'CE04OSPS': {
+        'long_name': 'Coastal Endurance, Oregon Offshore Cabled Shallow Profiler Mooring',
+        'tilt_correction': 0,
+        'colorbar_range': [-90, -50],
+        'vertical_range': [0, 200],
+        'deployed_depth': 200
+    },
     'CE06ISSM': {
-        'long_name': 'Washington Inshore Surface Mooring',
+        'long_name': 'Coastal Endurance, Washington Inshore Surface Mooring',
         'tilt_correction': 15,
         'colorbar_range': [-90, -50],
         'vertical_range': [0, 30],
         'deployed_depth': 29
     },
     'CE07SHSM': {
-        'long_name': 'Washington Shelf Surface Mooring',
+        'long_name': 'Coastal Endurance, Washington Shelf Surface Mooring',
         'tilt_correction': 15,
         'colorbar_range': [-90, -50],
         'vertical_range': [0, 87],
         'deployed_depth': 87
     },
     'CE09OSSM': {
-        'long_name': 'Washington Offshore Surface Mooring',
+        'long_name': 'Coastal Endurance, Washington Offshore Surface Mooring',
         'tilt_correction': 15,
         'colorbar_range': [-90, -50],
         'vertical_range': [0, 540],
         'deployed_depth': 542
     },
     'CP04OSSM': {
-        'long_name': 'Offshore Surface Mooring',
+        'long_name': 'Coastal Pioneer, Offshore Surface Mooring',
         'tilt_correction': 15,
-        'colorbar_range': [-150, 0],
+        'colorbar_range': [-90, -50],
         'vertical_range': [0, 450],
         'deployed_depth': 450
     },
     'CP03ISSM': {
-        'long_name': 'Inshore Surface Mooring',
+        'long_name': 'Coastal Pioneer, Inshore Surface Mooring',
         'tilt_correction': 15,
-        'colorbar_range': [-150, 0],
+        'colorbar_range': [-90, -50],
         'vertical_range': [0, 90],
         'deployed_depth': 90
     },
     'CP01CNSM': {
-        'long_name': 'Central Surface Mooring',
+        'long_name': 'Coastal Pioneer, Central Surface Mooring',
         'tilt_correction': 15,
-        'colorbar_range': [-150, 0],
+        'colorbar_range': [-90, -50],
         'vertical_range': [0, 130],
         'deployed_depth': 130
```

Comment:
Deployed depth and max vertical range should be 135

Reply:
PR updated.